ui/src/submissions/common/__tests__/__snapshots__/withFormItem.test.jsx.snap

```diff
@@ -11,16 +11,38 @@ exports[`withFormItem passes all props correctly to Form.Item and wrapped Input
       Error
     </span>
   }
-  label="Test Label"
+  label={
+    <Tooltip
+      arrowPointAtCenter={false}
+      autoAdjustOverflow={true}
+      mouseEnterDelay={0.1}
```

Comment:
do we need these values? Can we just use the default ones?

Reply:
They're default props from Ant's Tooltip, I'm not setting those values.

Reply:
if they are default why we set delay etc :/

Reply:
We don't. This is the snapshot and `0.1` is the default `mouseEnterDelay` set by Ant (https://ant.design/components/tooltip/#Common-API)

Reply:
so we don't have to explicity add it, by default it'll have this value :)

Reply:
We're not adding anything, note that in `withFormItem.jsx` we're only setting the title of the Tooltip. When running the jest tests the other values are added and if we remove them the tests fail 🙁